controllers/reconcile_commonaudit.go

```diff
@@ -73,6 +73,43 @@ func (r *CommonAuditReconciler) reconcileService(instance *operatorv1.CommonAudi
 	return reconcile.Result{}, nil
 }
 
+func (r *CommonAuditReconciler) reconcileZenService(instance *operatorv1.CommonAudit) (reconcile.Result, error) {
```

Comment:
With the latest design/architecture decisions, Common Audit Logging would take ownership of the zen-audit-svc Kubernetes Services ... and therefore the zen-audit-svc Kubernetes Services will be removed from Zen Operator etc.

Reply:
Noted.